### PR TITLE
[TFA] Fix the RHEL8 and RHEL9 images used in the config file

### PR DESCRIPTION
**Pull request by sayaleeraut**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by vamahaja** [comment]
@sayaleeraut are these changes working ?

**Review by MohitBis — APPROVED** [review]
LGTM

**Comment by sayaleeraut** [comment]
Execution logs - http://magna002.ceph.redhat.com/cephci-jenkins/saraut/PR_3718/

**Comment by sayaleeraut** [comment]
> @sayaleeraut are these changes working ?

Yes, please check the execution logs

File: conf/reef/upgrades/upgrade_8-node-cluster_mix_os.yaml [hunk 1]
```diff
@@ -2,7 +2,9 @@ globals:
   - ceph-cluster:
       name: ceph
       node1:
-        image-name: RHEL-8.6.0-x86_64-ga-latest
+        image-name:
+          openstack: RHEL-8.9.0-x86_64-ga-latest
+          ibmc: rhel-87-server-released
```
Comment (vamahaja) [inline conf/reef/upgrades/upgrade_8-node-cluster_mix_os.yaml:7]: IBM Cloud inventory needs to be changed, will share details.

Comment (sayaleeraut) [reply conf/reef/upgrades/upgrade_8-node-cluster_mix_os.yaml:7]: Okay, noted, once thats done, will send a PR to inculcate the changes.

Comment (vamahaja) [reply conf/reef/upgrades/upgrade_8-node-cluster_mix_os.yaml:7]: Use conf - `ibm-vpc-rhel-8.10-minimal-bx2-2x8`

Refer - https://github.com/red-hat-storage/cephci/pull/3756

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: conf/reef/upgrades/upgrade_8-node-cluster_mix_os.yaml (hunk 1 above)
Comment (sayaleeraut) [reply conf/reef/upgrades/upgrade_8-node-cluster_mix_os.yaml]: Done

**Review by vamahaja — APPROVED** [review]

**Pull request merged**